view/frontend/web/js/model/spi.js

```diff
@@ -119,6 +121,15 @@ define([
                         console.error('An unexpected PayPal error occurred', errors);
                         messageList.addErrorMessage({message: 'Warning: An unexpected error occurred. Please try again.'});
                     },
+                    'onClickPaymentOrder': async function () {
+                        try {
+                            return await onClickPaymentOrderCallback(pageSource);
+                        } catch (e) {
+                            console.error(e);
+                            fullScreenLoader.stopLoader();
+                            throw e;
```

Comment:
Does this get caught somewhere? Or are we just throwing uncaught errors into the console which will blow up the console?

Reply:
This will display to the console. We can catch the err here and not re-throw no problem. I was attempting to find a way to cleanly close the modal using err handling but it seems this is not a supported feature right now.